.editorconfig

```diff
@@ -0,0 +1,8 @@
+# Visual Studio generated .editorconfig file with C++ settings.
+root = true
+
+[*.{c++,cc,cpp,cppm,cxx,h,h++,hh,hpp,hxx,inl,ipp,ixx,tlh,tli}]
+
+# Visual C++ Code Style settings
+
+cpp_generate_documentation_comments = doxygen_triple_slash

```

Comment:
I tend to use and prefer the

```cpp
/** a thing.
 *  this thing does things.
 */
```

style instead of the triple slash one. I think it looks better! :slightly_smiling_face:

Reply:
thats what seems to be more of a standard anyways, I think VisualStudio is just weird....